addons/crm/static/src/views/forecast_search_model.js

```diff
@@ -3,15 +3,14 @@
 import { Domain } from "@web/core/domain";
 import { makeContext } from "@web/core/context";
 import { SearchModel } from "@web/search/search_model";
-
+import {
+  serializeDate,
+  serializeDateTime,
+} from "@web/core/l10n/dates";
```

Comment:
a new line should be added after this line, also optional but the line could be on one line, it's only 72 characters long on one line